README.md

```diff
@@ -255,7 +255,7 @@ To run a scan file upload using the CLI from docker:
 The commands are as follows:
 
 ```
-CMD_LINE_ARGS="upload /path/to/scan.zip  --springdoc.api-docs.enabled=false --spring.main.web-application-type=none"
+export CMD_LINE_ARGS="upload /tmp/scan.zip --springdoc.api-docs.enabled=false --spring.main.web-application-type=none"
```

Comment:
Should we keep /path/to/scan.zip?

Reply:
The docker commands are complicated. In this case, I'm specifying the path to the scan.zip inside the docker container, which has been mounted to the /tmp directory by the docker run command that follows the README:

```
docker run -d -v /path/to/file/AcmeRetail.yml:/config/ema.yml -v /path/to/scan.zip:/tmp  --env CMD_LINE_ARGS=${CMD_LINE_ARGS} --name event-management-agent event-management-agent:latest
```

The `/path/to/scan.zip` applies to the docker run command since that is the location of the scan.zip file on the user's computer, but it is mounted to `/tmp/scan.zip` inside the container, so `/tmp/scan.zip` is the correct location for the `export CMD_LINE_ARGS` command above.